### PR TITLE
EDGE-10056 | create_daily_campaign_activity_api

### PR DESCRIPTION
**Pull request by martin-hristov-acc**
**References**: [EDGE-10056](https://jira.gannett.com/browse/EDGE-10056)

**Code PR**: https://github.com/GannettDigital/reach-analytics-reporting-service/pull/1966

**Description**:
A new API for daily campaign activities

### CODE REVIEW

File: source/includes/_daily_campaign_activity.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+```
+curl -H "Authorization: Bearer Bearer OAUTH_ACCESS_TOKEN \
+https://api.localiqservices.com/client_reports/daily_campaign_activity/TEST_1?start_date=2016-12-01&end_date=2016-12-31"
+```
+
+>  Retrieve data for a specific campaign starting on a certain date
+
+```
+curl -g -H "Authorization: Bearer Bearer OAUTH_ACCESS_TOKEN \
```
Comment (brianroth-rl) [inline source/includes/_daily_campaign_activity.md:44]: This looks like a typo
```
Authorization: Bearer Bearer OAUTH_ACCESS_TOKEN \
```
Most likely this was copied from somewhere else, so I suspect it's incorrect there also.

File: source/includes/_daily_campaign_activity.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+|Field Name|Datatype|Description|
+|---|---|---|
+|report_type|String|Type of report|
+|report_date|String|Date report ran|
+|earliest_date_available|String|How far back data is available|
+|start_date|String|Start date of report|
+|end_date|String|End date of report|
+|bussiness_id|Integer|Business id|
+|bussiness_name|String|Business name|
```
Comment (brianroth-rl) [inline source/includes/_daily_campaign_activity.md:219]: typo

File: source/includes/_daily_campaign_activity.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+When using the GET method, the results can be filtered using these parameters:
+
+|Param|Function|
+|---|---|
+|start_date|Restricts the results to those occurring on or after this date|
+|end_date|Restricts the results to those occurring on or before this date|
+|global_master_campaign_id[]|Restrict results to one or more specific campaigns. This should be a comma separated string. Ex: global_master_campaign_id[]=TEST_1,TEST_2|
+|campaign_status[]|Restrict results to all campaigns with given status values.  Allowed values are running, stopped and ended. This should be a comma separated string. Ex: campaign_status[]=running,stopped|
+|campaign_cycle|Restrict results to a single campaign cycle|
```
Comment (brianroth-rl) [inline source/includes/_daily_campaign_activity.md:23]: Is this necessary, we don't call out cycles in the response so allowing this as a filter doesn't seem like something we want.

**Review by brianroth-rl — CHANGES_REQUESTED** [review]

File: source/includes/_daily_campaign_activity.md (hunk 3 above)
Comment (brianroth-rl) [reply source/includes/_daily_campaign_activity.md:23]: [It's not supported and should be removed](https://github.com/GannettDigital/reach-analytics-reporting-service/pull/1966/files#diff-f6bde42362fb05021213f3d538f637db596738a5a6c19df9e3af1d974468ae1bR1-R9)

**Review by brianroth-rl — APPROVED** [review]

**Review by brianroth-rl — APPROVED** [review]

**Comment by briannelson** [comment]
Can this be merged please?

**Pull request merged**